src/main/java/at/hannibal2/skyhanni/features/misc/massconfiguration/DefaultConfigFeatures.kt

```diff
@@ -39,7 +40,14 @@ object DefaultConfigFeatures {
                 "§eClick to run /shdefaultoptions!",
             )
         } else if (updated) {
-            val lastVersion = knownToggles.keys.last { it != SkyHanniMod.VERSION }
+            val lastVersion = knownToggles.keys.lastOrNull { it != SkyHanniMod.VERSION }
+                ?: ErrorManager.skyHanniError(
+                    "lastVersion is null, this should never happen",
+                    extraData = listOf(
+                        "knownToggles" to knownToggles,
+                        "version" to SkyHanniMod.VERSION,
+                    ).toTypedArray()
```

Comment:
```suggestion
                    "knownToggles" to knownToggles,
                    "version" to SkyHanniMod.VERSION,
```
The to typed array doesnt seem needed here